content/en/admin/config.md

```diff
@@ -1136,6 +1136,67 @@ Regeneration of home feeds is computationally expensive, if your Sidekiq is cons
 This setting has no relation to which users are considered active for the purposes of statistics, such as the Monthly Active Users number.
 {{</ hint >}}
 
+### Fetch All Replies {#fetch-all-replies}
+
+**Version history:**\
+4.x.x - added
+
+Fetch all replies fetches the tree of replies beneath an expanded post by recursively requesting the [replies collections](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-replies) of each of the statuses, and then requesting the status itself. Fetching replies is triggered by requesting the status's `context` - so will be triggered both from the web interface and external apps.
+
+Specifically, posts will be fetched if
+- The remote server correctly implements [ActivityPub/ActivityStreams Collections](https://www.w3.org/TR/activitypub/#collections), including [paging](https://www.w3.org/TR/activitystreams-core/#paging)
+- The remote server allows requests for replies collections to be made from the default instance actor - e.g. [`AUTHORIZED_FETCH`](#authorized-fetch) is not enabled
```

Comment:
s/`#authorized-fetch`/`#authorized_fetch`

Reply:
also is this correct framing? AUTHORIZED_FETCH is about using an http signature currently, which is done via the instance actor, so this line seems wrong

Reply:
fixed in https://github.com/mastodon/documentation/pull/1610/commits/56980ff9af3cedd761360457c9c364c68d8233ff